packages config for Python UDFs (#8805)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/core-upgrade/03-upgrading-to-v1.12.md

## New and changed features and functionality

### `packages` config for Python UDFs <Lifecycle status="beta" />

> You can specify public third-party PyPI packages for your Python UDF using the optional `packages` config. The warehouse installs these packages when creating the UDF, allowing your UDF to use functionality from external Python libraries. For more information, refer to [Defining UDFs in dbt](/docs/build/udfs#defining-udfs-in-dbt) and the [packages](/reference/resource-configs/packages) config reference page.

Reviewer: ```suggestion
You can specify public third-party PyPI packages for your Python UDF with the optional `packages` config. The warehouse installs these packages when it creates the UDF, which lets your UDF use functionality from external Python libraries. For more information, refer to [Defining UDFs in dbt](/docs/build/udfs#defining-udfs-in-dbt) and the [packages](/reference/resource-configs/packages) config reference.
```
